From 1b4317b2c96e1e40c7e3cefd0182d29829fed2da Mon Sep 17 00:00:00 2001
From: lintao <lintao@rock-chips.com>
Date: Wed, 24 Dec 2014 15:40:20 +0800
Subject: [PATCH] Revert "arm64: rockchip: rockchip_defconfig disable
 MMC_DW_IDMAC temporarily"

This reverts commit da9ae0c513b4a01ff4302a21a82746ddaeb2c16c.

Signed-off-by: lintao <lintao@rock-chips.com>
---
 arch/arm64/configs/rockchip_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index 6a0c8ee76c75..10309a76896e 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -511,6 +511,7 @@ CONFIG_MMC_PARANOID_SD_INIT=y
 CONFIG_MMC_BLOCK_MINORS=32
 # CONFIG_MMC_BLOCK_BOUNCE is not set
 CONFIG_MMC_DW=y
+CONFIG_MMC_DW_IDMAC=y
 CONFIG_MMC_DW_ROCKCHIP=y
 CONFIG_LEDS_GPIO=y
 CONFIG_LEDS_TRIGGERS=y
-- 
2.35.3

